docs/guides/deployment/bare_metal.rst

```diff
@@ -109,13 +109,29 @@ Save the file and exit, then restart the service.
 
 Set a password
 ==============
-There is no default password. Set a password by connecting from localhost.
+There is no default password. To set one, you will first need to get the Unix
+socket directory. You can find this by looking at your system.d unit file.
+
+.. code-block:: bash
+
+    $ sudo systemctl cat edgedb-server-3
+
+Look for a line starting with ``ExecStart=``. Get the value being passed for
+``--runstate-dir``. If that value contains ``%t``, replace that with ``/run``.
+So, ``%t/edgedb`` would become ``/run/edgedb``. Use that value in the next
+series of commands as shown.
```

Comment:
Ah I mean the `systemctl show ... | grep ... | awk ...` command does exactly what you described here, so that the user could just copy-paste and execute. If you feel explaining the command is reasonable, maybe we can move this explainer *after* the command into an optional block?

Reply:
Ah, right. I overlooked taking this out. Thanks for catching it!